### ISSUE TITLE
remove hbase docs directory to reduce image size.

### ISSUE THREAD

**Issue by Matrix0xCC** [PeterGrace/opentsdb-docker#28]
```
bash-4.4# du -sh *
212.0K	CHANGES.txt
4.0K	LEGAL
140.0K	LICENSE.txt
396.0K	NOTICE.txt
4.0K	README.txt
208.0K	bin
48.0K	conf
286.2M	docs
672.0K	hbase-webapps
86.3M	lib
8.0K	logs
```

**Comment by PeterGrace** [comment]
Good lord that's a lot of docs.  I agree 100%.

**Comment by PeterGrace** [comment]
Fixed in 0444900, thanks again!
